packages/dynamite/dynamite/lib/src/openapi_builder.dart

```diff
@@ -78,16 +78,45 @@ class OpenAPIBuilder implements Builder {
 
       final state = State(buildConfig);
 
-      // Imports need to be generated after everything else so we know if we need the local part directive,
-      // but they need to be added to the beginning of the output.
-      final output = ListBuilder<Spec>()
-        ..addAll(generateClients(spec, state))
-        ..addAll(generateSchemas(spec, state))
-        ..addAll(buildOfsExtensions(spec, state))
-        ..addAll(buildSerializer(state))
-        ..insertAll(0, generateImports(outputId, state));
+      final output = Library((final b) {
+        final analyzerIgnores = state.buildConfig.analyzerIgnores;
+        if (analyzerIgnores != null) {
+          b.ignoreForFile.addAll(analyzerIgnores);
+        }
+
+        b
+          ..name = toLibraryName(p.basenameWithoutExtension(inputId.path))
```

Comment:
The `_openapi` part of the resulting library names should be dropped.

Reply:
I fear that we potentially get naming conflicts if we do so.

Reply:
And it is needed to avoid reserved names to be used.

`enum` and `$enum` are not allowed but `enum_openapi` is. We face this in the e2e specs